Makefile

```diff
@@ -33,6 +33,9 @@ test:
 	poetry run pytest -m "not perf" --ignore tests/e2e --cov=aws_lambda_powertools --cov-report=xml
 	poetry run pytest --cache-clear tests/performance
 
+test-pydanticv2:
+	poetry run pytest -m "not perf" --ignore tests/e2e
```

Comment:
Feel free to also pass only e2e tests we know Pydantic v2 could be used

Reply:
We don't normally run e2e on our CI mainly because it takes a long time to run. Are you sure you want me to add it here?

Reply:
My bad!!!! I read when I was too hungry and missed the "ignore".

Let's roll with it. The boost in Perf would be

- only run this workflow IF files/features related to Pydantic v2 are changed
- run tests selectively eg tests/.../parser, event handler etc